Sources/URBNSwiftyConvenience/UIView+Additions.swift

```diff
@@ -158,4 +158,35 @@ extension UIView {
         // Alert the layout needs updating
         setNeedsLayout()
     }
+    
+    func sizeToWidthOfContainer(){
+        sizeToWidthOfContainerWithEqualInset(0)
+    }
+    
+    func sizeToWidthOfContainer(WithEqualInset inset: CGFloat){
```

Comment:
small `with`

Reply:
Doh! That was a stupid thing for me to overlook. Thanks for catching it. Fixed.